[2025-04-20T17:34:15.284+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-20T17:34:15.339+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supply_chain_dag.collect_data scheduled__2025-04-20T16:00:00+00:00 [queued]>
[2025-04-20T17:34:15.364+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supply_chain_dag.collect_data scheduled__2025-04-20T16:00:00+00:00 [queued]>
[2025-04-20T17:34:15.365+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-20T17:34:15.402+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): collect_data> on 2025-04-20 16:00:00+00:00
[2025-04-20T17:34:15.414+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=968) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-20T17:34:15.417+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'supply_chain_dag', 'collect_data', 'scheduled__2025-04-20T16:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/supply_chain_dag.py', '--cfg-path', '/tmp/tmp2fwhm97m']
[2025-04-20T17:34:15.420+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask collect_data
[2025-04-20T17:34:15.420+0000] {standard_task_runner.py:72} INFO - Started process 969 to run task
[2025-04-20T17:34:15.529+0000] {task_command.py:467} INFO - Running <TaskInstance: supply_chain_dag.collect_data scheduled__2025-04-20T16:00:00+00:00 [running]> on host 4eea4bee2aed
[2025-04-20T17:34:15.708+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supply_chain_dag' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-20T16:00:00+00:00'
[2025-04-20T17:34:15.710+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-20T17:34:15.817+0000] {logging_mixin.py:190} INFO - {'status': 'Data sent to Kafka', 'data': {'order_id': 'ORD-7418FB71', 'departure_date': '2025-04-17', 'arrival_date': '2025-04-27', 'quantity': 281, 'warehouse': 'WH2', 'destination': 'DistributionCenter', 'shipping_mode': 'Second Class', 'order_amount': 4302.38, 'customer_id': 'CUST-B456D0C7', 'late_delivery_risk': 1}}
[2025-04-20T17:34:15.818+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-20T17:34:15.836+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-20T17:34:15.837+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=supply_chain_dag, task_id=collect_data, run_id=scheduled__2025-04-20T16:00:00+00:00, execution_date=20250420T160000, start_date=20250420T173415, end_date=20250420T173415
[2025-04-20T17:34:15.925+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-20T17:34:15.960+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-20T17:34:15.962+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
